### PR TITLE
HeightMapTessellator with WebMercatorTilingScheme

### PR DESCRIPTION
**Pull request by pasu**
Hi all,
I open an issue here:
https://github.com/AnalyticalGraphicsInc/cesium/issues/5779
and this is my modification for it
Hope you can consider about my understanding, thanks.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
@pasu, thanks for the pull request!

I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

__I am a bot who helps you make Cesium awesome!__ Thanks again.

**Comment by pjcozzi** [comment]
Thanks for the contribution, @pasu!  Could you please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) so we can review this?

@bagnell can you review when ready?

**Comment by pjcozzi** [comment]
@pasu also please check the eslint failures in CI: https://travis-ci.org/AnalyticalGraphicsInc/cesium/builds/268256371?utm_source=github_status&utm_medium=notification

**Comment by pjcozzi** [comment]
We now have a CLA from @pasu, thanks again!

**Comment by pasu** [comment]
I got this CI procedure, and fixed the test unit, please review that.
Thanks for the review @bagnell

**Comment by pjcozzi** [comment]
Thanks for the updates, @pasu.  @bagnell can you please review this when you have a chance?  It could be after the 1.37 release tomorrow.

**Comment by pasu** [comment]
Hi, all @bagnell 
Could you please review this and share the comment?
I have another request and wait to pull
Thanks :laughing:

File: Source/Core/OrientedBoundingBox.js [hunk 1]
```diff
@@ -272,8 +272,8 @@ define([
         if (!defined(rectangle)) {
             throw new DeveloperError('rectangle is required');
         }
-        if (rectangle.width < 0.0 || rectangle.width > CesiumMath.PI) {
-            throw new DeveloperError('Rectangle width must be between 0 and pi');
+        if (rectangle.width < 0.0 || rectangle.width > 2*CesiumMath.PI) {
```
Comment (hpinkos) [inline Source/Core/OrientedBoundingBox.js]: Use `CesiumMath.TWO_PI`

**Comment by pasu** [comment]
hi, @hpinkos 
Thanks for your review, I fixed and checked it.

**Comment by hpinkos** [comment]
@bagnell do you have time to take a look at this?

**Comment by bagnell** [comment]
Looks good, thanks @pasu!

**Comment by pasu** [comment]
It's my honor. Thanks for review.

**Pull request merged**